pkg/apis/flowcontrol/install/install.go

```diff
@@ -38,6 +39,11 @@ func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(flowcontrolv1beta1.AddToScheme(scheme))
 	utilruntime.Must(flowcontrolv1beta2.AddToScheme(scheme))
```

Comment:
is there a reason to continue to add v1beta1 and v1beta2 to the legacy scheme? since  v1beta1 has been removed in 1.26, and v1beta2 will be removed in 1.29?

Reply:
Yes, the apiserver has to be able to read old stuff from storage; Kubernetes in general has no limit on how outdated is the version of an object in storage.

Reply:
correct, we will continue supporting decode from v1beta1 and v1beta2 indefinitely until automated storage migration is built